CauldronMods/Controller/Heroes/Cricket/Cards/SwarmingFrequencyCardController.cs

```diff
@@ -12,6 +12,7 @@ public class SwarmingFrequencyCardController : CardController
         public SwarmingFrequencyCardController(Card card, TurnTakerController turnTakerController) : base(card, turnTakerController)
         {
             AllowFastCoroutinesDuringPretend = false;
+            SpecialStringMaker.ShowNumberOfCardsInPlay(new LinqCardCriteria(c => c.IsEnvironmentTarget, "", useCardsSuffix: false, singular: "environment target", plural: "environment targets"));
```

Comment:
Would we instead want to display the environment target with the lowest hp, if one exists?

Reply:
Yeah that works well. I would do this with two special strings [these might be slightly incorrect names and stuff, as this is from memory] :

1) SpecialStringMaker.ShowEnvironmentTargetWithLowestHP().Condition = () => FindCardsWhere(c => c.IsEnvironmentTarget).Count()> 0

2)SpecialStringMaker.ShowSpecialString(() => "There are no environment targets in play").Condition = () => FindCardsWhere(c => c.IsEnvironmentTarget).Count() == 0